test/test_integration_resource.py

```diff
@@ -0,0 +1,182 @@
+               ['user', 'rotate-api-key', '-i', 'admin'], exit_code=1)
+        self.assertIn("500 Server Error", attempt_to_rotate_admin_key)
+
+    def test_user_rotate_api_key_without_flag_returns_error(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['user', 'rotate-api-key', 'someinput'], exit_code=1)
+        self.assertIn('Error', self.capture_stream.getvalue())
+
+    def test_user_rotate_api_key_flag_without_value_returns_error(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['user', 'rotate-api-key', '-i'], exit_code=1)
+
+        self.assertIn('Error', self.capture_stream.getvalue())
+
+    def test_user_change_password_does_not_provide_password_prompts_input(self):
+        with patch('getpass.getpass', side_effect=['', 'Mypassw0rD2\!']):
+            output = self.invoke_cli(self.cli_auth_params,
+                          ['user', 'change-password'])
+        self.assertIn("Password has been successfully changed", output)
+
+    def test_user_change_password_not_complex_enough_prompts_input(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                      ['user', 'change-password', '-p', 'someinvalidpassword'], exit_code=1)
+        self.assertIn("Invalid password. Please verify that it has the correct password complexity", output)
+
+    def test_user_change_password_meets_password_complexity(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                      ['user', 'change-password', '-p', 'Mypassw0rD2\!'])
+        self.assertIn("Password has been successfully changed", output)
+
+    def test_user_change_password_empty_password_provided_prompts_input(self):
+        with patch('builtins.input', side_effect=['', '']):
+            output = self.invoke_cli(self.cli_auth_params,
+                          ['user', 'change-password'], exit_code=1)
+        self.assertIn("Invalid password. Please verify that it has the correct password complexity", output)
+
+    @integration_test
+    def test_host_rotate_api_key_without_host_prompts_input(self):
+        with patch('builtins.input', side_effect=['somehost']):
+            output = self.invoke_cli(self.cli_auth_params,
+                   ['host', 'rotate-api-key'])
+            self.assertIn("API key for 'somehost' was successfully rotated", output)
+
+    @integration_test
+    def test_host_rotate_api_key_with_host_provided_rotates_host_api_key(self):
+        output = self.invoke_cli(self.cli_auth_params,
+               ['host', 'rotate-api-key', '-i', 'somehost'])
+        self.assertIn("API key for 'somehost' was successfully rotated", output)
+
+    def test_host_rotate_api_key_without_flag_returns_error(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                        ['host', 'rotate-api-key', '-i', 'someinput'], exit_code=1)
```

Comment:
you need to remove the flag from here

Reply:
why do I need to remove the flag?

Reply:
The name of the test **without_flag_returns_error**